### DISCUSSION TITLE
Launch On Startup & Start Minimised

### DISCUSSION THREAD

**Discussion by trymeouteh** [podman-desktop/podman-desktop#5590] [Ideas]
Please add two settings to Podman Desktop...

Launch on Startup: A setting on weather to launch Podman Desktop during the system startup

Start Minimised: Weather Podman Desktop will launch mininized to the system tray on startup or open the Podman Desktop window on startup

**Comment by trmendes** [comment]
![login](https://github.com/containers/podman-desktop/assets/16507629/5e891229-2d53-4bee-8fd4-1d8304e0ca41)

@trymeouteh If I understood it correctly you would like to have configured Podman-Desktop to start with your system and when that happen you would like to have it minimized to the systray. I've such configuration running on my macbook and I made it thought the settings marked on the screenshot above.

**Reply by be-michael-tesar** [reply]
I have the the exact same settings in 1.17.2 on macOS 15.4 and it starts on login but not minimized. Maybe I should report it as a bug?

**Comment by BK0STAR** [comment]
same bug. It don't work

**Comment by ArturM94** [comment]
The Linux version which published on [Flathub](https://flathub.org/en/apps/io.podman_desktop.PodmanDesktop) lacks these settings

<img width="1918" height="1052" alt="image" src="https://github.com/user-attachments/assets/3f79fe02-30e0-4679-b237-bc2060bfc8df" />

**Comment by be-michael-tesar** [comment]
Podman 5.7.1 on macOS 26.2 stopped starting at the system start...
